src/sandbox/wiktor/HistoryTable.tsx

```diff
@@ -0,0 +1,106 @@
+import CallMadeIcon from '@mui/icons-material/CallMade';
+import CallReceivedIcon from '@mui/icons-material/CallReceived';
+import Chip from '@mui/material/Chip/Chip';
+import Paper from '@mui/material/Paper';
+import { DataGrid, GridColDef, GridRenderCellParams } from '@mui/x-data-grid';
+import React from 'react';
+import styled from 'styled-components'; // useTheme
+
+import { HistoryTableData } from './sampleHistoryTableData';
+
+const Container = styled.div`
+  max-width: 800px;
```

Comment:
Możemy pozwolić tabeli na width: 100%, wiem, że wtedy jest dużo pustego miejsca, ale jest fajnie responsywna + jak przeniesiesz do Statistics to po lewej jest Drawer, który zabiera trochę szerokości :D 

```suggestion
```

Reply:
corrected